lib/log_formatter.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Bark.LogFormatter do
+  @moduledoc """
+  To use in an application, add this in `config/`
+
+  ```elixir
+  config :logger, :console, format: "..."
+
+  # Pulls the format string from existing console logger, or uses a default
+  config :logger, :default_formatter,
+      format: {Bark.LogFormatter, :format},
+  ```
+
+  https://hexdocs.pm/logger/1.17.3/Logger.Formatter.html#module-formatting-function
+  """
+
+  @default_format "$time $metadata[$level] $message\n"
+  @format :logger
+          |> Application.compile_env(:console, [])
+          |> Keyword.get(:format, @default_format)
+
+  @spec format(atom, IO.chardata(), Logger.Formatter.date_time_ms(), keyword()) :: IO.chardata()
+  def format(level, message, timestamp, metadata) do
+    metadata = metadata |> format_metadata() |> escape_values()
+
+    Logger.Formatter.format(
+      Logger.Formatter.compile(@format),
+      level,
+      message,
+      timestamp,
+      metadata
+    )
+  end
+
+  # In order to not break existing DataDog log queries for logs in DataDog,
+  # we have to match the same log/metadata format as defined in Bark.parse_message
+  # we manually
+  defp format_metadata(metadata) do
+    case Keyword.get(metadata, :mfa) do
+      {module, function, arity} ->
+        module = module |> Atom.to_string() |> String.replace("Elixir.", "")
```

Comment:
```suggestion
        module = inspect(module)
```

Inspecting will handle pretty printing the module name without the Elixir domain

```elixir
iex(1)> to_string Test
"Elixir.Test"
iex(2)> inspect Test
"Test"
```

Reply:
Good call; I just copied the String replacement verbatim from the original Bark module; will remove!